doc/api/http.md

```diff
@@ -2131,6 +2131,11 @@ Sends a HTTP/1.1 100 Continue message to the client, indicating that
 the request body should be sent. See the [`'checkContinue'`][] event on
 `Server`.
 
+### `response.writeEarlyHints()`
```

Comment:
This is missing parameters documentation.

Reply:
updated, thanks!